tests/example_app/example/about/templates/about.html

```diff
@@ -0,0 +1,18 @@
+{% extends "layout.html" %}
+
+{% block body %}
+  <br/>
+  <br/>
+  <a href="/about/our-story">Our Story</a>
```

Comment:
Can you make all of the rename suggestions from a previous comment and also use url_for instead of hard coding the URL?